src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/GenerateApplicant1NoticeOfProceeding.java

```diff
@@ -105,8 +110,15 @@ private void generateJointNoticeOfProceedings(CaseData caseData, Long caseId) {
         } else {
             log.info("Generating applicant 1 notice of proceedings for joint case id {} ", caseId);
 
-            content = jointTemplateContent.apply(caseData, caseId, caseData.getApplicant1(), caseData.getApplicant2());
-            templateId = NFD_NOP_JA1_JOINT_APP1APP2_CIT;
+            if (caseData.getIsJudicialSeparation().toBoolean() && caseData.getApplicant1().isApplicantOffline()) {
```

Comment:
Let's confirm where ever we use this, we don't just call to get the variable itself. As we need to confirm how NULL will behave.